.travis.yml

```diff
@@ -22,7 +22,9 @@ matrix:
           packages:
             - clang-3.8
 
-    - env: COMPILER=clang++-7
+    - env:
+      - COMPILER=clang++-7
+      - EXERCISM_COMMON_CATCH=false
```

Comment:
Looks good. `EXERCISM_COMMON_CATCH` is enabled by default.
We use the fastest compiler to test the students' setup.